README.md

```diff
@@ -1,2 +1,159 @@
+
+
 # Todo-List
 Simple TODO List project using webpack and ES6 Syntax
+
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Awesome books</b></h3>
+This repository is for the activity Module 2 week1 "Awesome books: plain JavaScript with objects".
+In this project, we built a basic website that allows users to add/remove books from a list. we achieved it that by using JavaScript objects and arrays to dynamically modify the DOM and add basic events.
```

Comment:
- [ ] Kindly modify your README file to have content related to the Todo List app as it's the app you are currently working on and not Awesome Books. Please edit the whole README file 👍